centaur/src/main/resources/standardTestCases/invalidate_bad_caches/invalidate_bad_caches_no_copy.wdl

```diff
@@ -1,5 +1,8 @@
 task make_file {
     Boolean ready = true
+
+    # This seed ensures that we would only ever call cache within the same instance of this workflow.
```

Comment:
To be honest, I think just making this file "different" from the `invalidate_bad_caches.wdl` to avoid cross-call-cache mixing was all that was needed... 🤷‍♂️

Reply:
Care to try that more minimal change first?

Reply:
Yeah, I think that's a good idea, which would normally just be "use the first commit" - but I do want to add the "make sure it would normally call cache before making sure we can invalidate it" sanity check